e2e/mock-api/src/app/routes.auth.js

```diff
@@ -289,6 +291,12 @@ export default function (app) {
       } else {
         res.status(406).send('Middleware header is missing.');
       }
+    } else if (req.path.includes('tokens-expiring-soon')) {
+      const tokensExpiringSoon = { ...oauthTokensExpiringSoon, access_token };
+      res.json(tokensExpiringSoon);
+    } else if (req.path.includes('tokens-expired')) {
+      const tokensExpired = { ...oauthTokensExpired, access_token };
+      res.json(tokensExpired);
```

Comment:
Just to make sure, object spreads evaluate from left to right. So, if the `access_token` object and the `oauthTokensExpir*` object being spread contain the same property, `access_token` will currently have priority. Is that what you're intending here? If `access_token` is the default object, and `oauthTokensExpi*` is the "modifying" object, then I'd put `access_token` first (left).

Reply:
Yes that is the intention, for consistency with the rest of the function that returns the regular **oauthTokens** response with access_token replacing the value within the oauthResponse. As per the above conversation, it looks like v4() replaces the value of the access token. Does this sound correct to you? Or am I replacing it when I don't need to?

Reply:
Yes, you are correct. I don't know why, but I confused `access_token` with a tokens object, rather than what it is, a string value. Sorry!